src/tp210.jl

```diff
@@ -0,0 +1,25 @@
+# TP problem 210 in NLS format
+#
+#   Source:
+#   Klaus Schittkowski,
+#   More test examples for nonlinear programming codes,
+#   Lecture Notes in Economics and Mathematical Systems 282,
+#   Springer Verlag Berlin Heidelberg, 1987
+#   10.1007/978-3-642-61582-5
+#
+# A. Montoison, Montréal, 06/2018.
+
+export tp210
+
+"Test problem 210 in NLS format"
+function tp210()
+
+  nls = Model()
+  x0  = [-1.2; 1]
+  @variable(nls, x[i=1:2], start=x0[i])
+
+  @NLexpression(nls, F1, 1000 * (x[2] - x[1]^2))
```

Comment:
We probably can define tp207 to tp210 using an argument
```julia
function tp207to210(a = 1)
  ...
  @NLexpression(nls, F1, a * (x[2] - x[1]^2))
  ...
end

tp207() = tp207to210(1)
tp208() = tp207to210(10)
...
```
Possibly tp211 as well, though it doesn't seem as interesting.